highest_number_cubed/main.py

```diff
@@ -1,5 +1,9 @@
 def highest_number_cubed(limit):
-    pass
+    for num in range(limit): 
+        if num % 3 == 0:
+         num = max(range(limit)) 
+         root = num ** (1./3) 
+    return int(root)     
 
 
```

Comment:
Only issue I see is with certain numbers. 0 will error and the function is returning the cube root of < than the limit. So 1 returns 0 instead of 1, 64 returns 3 instead of 4, etc although I don't see any instructions specifying if we are supposed to be doing < limit or <= limit and none of the tests dealt tested it.